src/modules/admin/admin.controller.js

```diff
@@ -0,0 +1,70 @@
+const Admin = require("./admin.model.js");
+
+// check admin exsists or not for given email
+const checkAdminExsists = async (email) => {
+  try {
+    const adminExsists = await Admin.findOne({ where: { email: email } });
+    return adminExsists;
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+// function for register admin in system.
+const registerAdmin = async (req, res) => {
+  const data = { ...req.body };
+  const checkAdmin = await checkAdminExsists(data.email);
+  if (!checkAdmin) {
```

Comment:
please try to avoid so many nested if else block.

Reply:
Ok I will